plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/util/SelectFirstMatchingPrefixListener.java

```diff
@@ -54,60 +47,33 @@ public void setContents(SortedSet<String> contents) {
   public void modifyText(ModifyEvent event) {
     String lastAsOfEvent = lastString;
     String prefix = targetCombo.getText();
-    try {
-      lastString = prefix;
-      // If content was deleted, don't immediately replace it
-      if (prefix.length() < lastAsOfEvent.length()) {
-        return;
-      }
-      // Don't autofill if the user isn't at the end of the string
-      if (targetCombo.getCaretPosition() < prefix.length() || prefix.length() < 6) {
-        return;
-      }
-      SortedSet<String> matchesAndLater = contents.tailSet(prefix);
-      if (matchesAndLater.isEmpty()) {
-        return;
-      }
-      String firstMatch = matchesAndLater.first();
-      if (!firstMatch.startsWith(prefix) || firstMatch.equals(prefix)) {
-        return;
-      }
-      int comboIndex = contents.size() - matchesAndLater.size();
-      // Don't fire on this modification of the combo
-      targetCombo.removeModifyListener(this);
-      targetCombo.select(comboIndex);
-
-      // Select the text that was added
-      targetCombo.setSelection(new Point(prefix.length(), firstMatch.length()));
 
-      // Fire on later user modifications of the combo
-      targetCombo.addModifyListener(this);
-    } finally {
```

Comment:
Here, I just removed the `try-finally` block and de-indented code, but apparently, the GitHub diff doesn't highlight this in an effectively.